test/utilities/miscellaneous.ts

```diff
@@ -60,37 +60,6 @@ export async function findElementByText(
   return element!;
 }
 
-/**
- * @param operation identifies if it's a pull or push operation
- * @param changes indicates if changes are expected or not
- * @param type indicates if the metadata is expected to have been created, changed or deleted
- * @returns the output panel text after
- */
-export async function verifyPushAndPullOutputText(
```

Comment:
You're correct that this should belong in pushAndPull.e2e.ts instead. 🙂

Reply:
When I first read this comment I was so confused about it because in develop I see it in `pushAndPull.e2e.ts` already, and also I couldn't see some of the recent refactoring I did, and then I realized the base branch in this PR is `testesm` and not `develop`, so some of the latest changes are not here. @peternhale could you please update it to see actual differences? 🙂

Reply:
I'll look

Reply:
I'll move this function to pushAndPull and then merge develop to my branch

Reply:
I was referring to this 
<img width="692" alt="image" src="https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/113132642/a40f3a16-b922-4230-a2ab-ab2245962db2">

Reply:
@CristiCanizales This PR only fixes the flappers for AnInitialSuite.e2e.ts.  If we merge it into develop, then all the other E2E tests would start failing because fixing their flappers isn't within the scope of this PR.  We want to have ESM as a parallel branch to develop, with this being the first commit in that parallel branch.

Reply:
then shouldn't we update testesm with develop first?

Reply:
@CristiCanizales Makes sense - so first merge testesm with develop to get the new features, then merge testesm into testesm_pete.

Reply:
This is done